### ISSUE TITLE
False positive: "Color-contrast" rule fails when using particular text-shadow values

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2541]
```color-contrast``` rule fails when an element has text-shadow values as mentioned in Case-1 in the example. 

Demo link: https://codepen.io/iamrafan/pen/abNevdV?editors=1100

<pre><code>axe-core version: 4.0.2
axe version: 4.6.1</code></pre>

**Comment by padmavemulapati** [comment]
Verified with the latest axe-core develop branch code(v4.0.2 dated 12/11/2020)
with the code snippets: color-contrast - for text-shadows is passing for the below three cases.
1. `<div id="target" style="background-color: #666; color:#aaa; 
				text-shadow: 0 0 0.09em #000, 0 0 0.09em #000, 0 0 0.09em #000;">
				Hello world
				</div>`
2. `<div id="target" style="background-color: #aaa; color:#666; 
				text-shadow: 0 0 0.09em #000, 0 0 0.09em #000, 0 0 0.09em #000;">
				  Hello world
				</div>`
3.`<div id="target" style="background-color: #aaa; color:#666; 
				text-shadow: 0 0 0.09em #000, 0 0 0.09em #000, 0 0 0.09em #000;">
				Hello world
				</div>`
![image](https://user-images.githubusercontent.com/47409579/98974069-7ff49b80-253a-11eb-8ef0-c613bce12945.png)


and failing for the following two cases:
1.`<div id="parent" style="height: 40px; width: 30px; background-color: #000;">
					<div id="target" style="height: 20px; width: 15px; text-shadow: red 0 0 0.05em">foo
					</div></div>`
2.`<div style="height: 40px; width: 30px; background-color: #800000;">
						<div id="target" style="height: 20px; width: 15px; text-shadow: red 0 0 1em, green 0 0 0.5em">foo
						</div></div>`

![image](https://user-images.githubusercontent.com/47409579/98974135-913da800-253a-11eb-98d6-6d1b3196b7a0.png)
